site/app/templates/misc/SourceSettings.twig

```diff
@@ -13,7 +13,7 @@
     else {
         editor{{ number }}.setSize("100%", "auto");
     }
-    var theme = localStorage.getItem("codeDisplayStyle");
+    var theme = localStorage.getItem("theme");
```

Comment:
Will this properly account for `window.matchMedia("(prefers-color-scheme: dark)").matches` as well?